test/logger.test.js

```diff
@@ -9,33 +9,36 @@ const { ecsMeta } = require('../utils');
 const chance = new Chance();
 
 describe('#setupNodeLogger()', () => {
+  let msg;
+  beforeEach(() => {
+    msg = chance.string({ symbols: false, alpha: true });
+  });
+
   describe('#json', () => {
     it('should printout the configured severity level', () => {
       const log = setupNodeLogger({ json: true, level: 'warn' });
 
-      let printout = chance.string();
+      // let msg = chance.string();
```

Comment:
Commented out code should definitely not be committed...

Reply:
dangling commennt